src/main/kotlin/main.kt

```diff
@@ -1,4 +1,10 @@
 fun main(args: Array<String>){
-    val test = "lalala"
-    println("$test")
+    val checkersModel = CheckersModel()
```

Comment:
It's good that you try to test it, but don't use `main` for this purpose. Please add some JUnit tests for it.